packages/plugins/rudder_plugin_interface/test/src/models/utils_test.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:test/test.dart';
+import 'package:rudder_sdk_flutter_platform_interface/src/utils.dart';
+
+void main() {
+  test('conversion of datestring object to string in format YY-mm-dd', () {
```

Comment:
The format is `yy-MM-dd` as per previous implementation. Have we changed it now?

Reply:
no, typo in the test description

Reply:
fixed it